cmd/talisman/run_px_upgrade.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
```

Comment:
I like this, but is this supposed to be delivered with talisman? I feel talisman could do this also without having a script. Maybe we can add it in a future change.